app/src/main/java/com/amaze/filemanager/ui/fragments/FtpServerFragment.kt

```diff
@@ -571,6 +568,11 @@ class FtpServerFragment : Fragment(R.layout.fragment_ftp) {
         }
         port.text = "${resources.getString(R.string.ftp_port)}: $defaultPortFromPreferences"
         updatePathText()
+
+        if(defaultPathFromPreferences == FtpService.defaultPath(requireContext()))
```

Comment:
add braces.

Reply:
> add braces.

I definitely can do, but why? This looks clean, IMO. Since it is just a single statement.

Reply:
> I definitely can do, but why? This looks clean, IMO. Since it is just a single statement.

Google guidelines for code in android and consistency with the codebase.